jenkins-integration/beaker/install/shared/30_customize_settings.rb

```diff
@@ -0,0 +1,25 @@
+require 'json'
+
+test_name 'Configure settings on SUT'
+
+def get_puppet_settings_from_env()
+  ENV['PUPPET_GATLING_PUPPET_SETTINGS']
+end
+
+puppet_settings = JSON.parse(get_puppet_settings_from_env())
+
+Beaker::Log.notify("INITIALIZING PUPPET SETTINGS to: #{puppet_settings}")
+
+sections = puppet_settings.keys
+sections.each do |s|
+  Beaker::Log.notify("Updating puppet settings for section '#{s}'")
+  puppet_settings[s].each do |setting, value|
+    Beaker::Log.notify("In section '#{s}', setting '#{setting}' to '#{value}'")
+    on(master, puppet("config set #{setting} #{value} --section #{s}"))
+  end
+end
+
+puppet_conf = on(master, "cat /etc/puppetlabs/puppet/puppet.conf").stdout.chomp
```

Comment:
that would be better but then I'd have to test it again :)  And I'm assuming we probably aren't going to be doing much (any?) additional testing on p3?